packages/components/src/styles/base.css

```diff
@@ -233,6 +233,11 @@ Use the modern Firefox focus style for all focusable elements.
     outline: auto;
   }
 
+  /* Except for Codemirror! */
+  :-moz-focusring.cm-content {
+    outline: none;
+  }
```

Comment:
Is this part not possible in `EditorView.theme`? A bit strange to have them in different places.

Cursory look at https://github.com/marijnh/style-mod#documentation makes me think that it should be possible, since even patterns such as `{"@media screen and (min-width: 400px)": {...}}` are supported.